contents/calendar/_index/main.markdown

```diff
@@ -25,6 +25,21 @@ events (excluding HackerspaceSG’s cleaning every Tuesday, of course).
 
 # Hosting Events
 
+## COVID-19 measures
+
+In view of the current outbreak of the Coronavirus Disease 2019 (COVID-19), HackerspaceSG strongly encourages event organisers to take the following measures:
+
+* Ask visitors to fill out a travel and health declaration form, and to include their personal information for contact tracing purposes, before the event.
+* Attendees who have been to mainland China, Iran, northern Italy, the Republic of Korea, or [any other such regions highlighted by the Singapore Ministry of Health](https://www.gov.sg/article/covid-19-travel-restrictions-for-foreign-visitors-entering-singapore), within the last 14 days, should be asked to not participate in the event.
+* Attendees who exhibit mild respiratory infection symptoms should wear a mask at all times during the event.
+* Attendees who exhibit serious respiratory infection symptoms should be asked to leave the HackerspaceSG premises.
+* Full discretion falls on either the event organiser, or any other HackerspaceSG member in the premises at the time, to enforce this.
+* Handshaking and other forms of physical contact should be discouraged.
+
+We encourage everyone to be vigilant, socially responsible, and observe good personal hygiene. Please seek [medical assistance](https://www.flugowhere.gov.sg/?searchValue=208576&tab=0) promptly if you are unwell.
+
+Thank you for your cooperation and understanding.
+
```

Comment:
In view of the fluid changes of COVID-19, we should add a last updated date for this section. Let's be frank about this, an average website visitor will not do a git blame to see when the section was last updated.

Reply:
Added: https://github.com/hackerspacesg/hackerspace.sg/commit/06db371d9b5a6754bf2bb996e98d45c37a9f0a7d#diff-c3781d18a09c98f50f516cb3129cd250R30